dbt_semantic_interfaces/parsing/where_filter/parameter_set_factory.py

```diff
@@ -49,6 +49,7 @@ def create_time_dimension(
         time_granularity_name: Optional[str] = None,
         entity_path: Sequence[str] = (),
         date_part_name: Optional[str] = None,
+        custom_granularity_names: Sequence[str] = (),
```

Comment:
Should this not be optional?

Reply:
felt better for this to be default to a sequence, so we don't need to handle this at the bottom where we do
```python
StructuredDunderedName.parse_name(
            name=time_dimension_name, custom_granularity_names=custom_granularity_names or ()
        )
```

Reply:
Yeah I guess I just want to make sure people don't skip it without realizing that it's necessary. Like sometimes I like to leave params like this as required so that people are forced to actively pass in the empty value and hopefully in the process they check out what it's used for and make sure they're doing the right thing.

Reply:
Ooo yea, ill drop the default